Sources/Apollo/Cancellable.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// An object that can be used to cancel an in progress action.
+public protocol Cancellable: class {
+    /// Cancel an in progress action.
+    func cancel()
+}
+
+// MARK: - URL Session Conformance
+
+extension URLSessionTask: Cancellable {}
+
+// MARK: - Early-Exit Helper
+
+/// A class to return when an error that should cause us to bail out of something still needs to return `Cancellable`.
+public final class ErrorCancellable: Cancellable {
```

Comment:
I'm wondering if this should really be tied to errors, or whether something more generic like an `EmptyCancellable` or  `NullCancellable` (from [null object pattern](https://en.wikipedia.org/wiki/Null_object_pattern)), would make sense.

Reply:
Ooh good call - `EmptyCancellable` is way more accurate